### DISCUSSION TITLE
Should move work in pieces?

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#3918] [General]
I'm running a big old `beet mv` on my whole library. 20,000 files. With `-vv` enabled it seems that beets is coming up with a list of items to move, which it will then iterate through to move.

In practice, this means waiting 5-30 minutes before a single move takes place. Is there a more real-time method that would be good in this situation?

Or are my plugins to blame? I see `the` logging every band name yet no moves are taking place (yet).

the inline fetchart orig_date convert mbsync duplicates replaygain info fromfilename chroma

https://github.com/beetbox/beets/blob/master/beets/ui/commands.py#L1118

**Comment by sampsyo** [comment]
That's odd! I don't have a guess at what the root cause might be. To test the plugin hypothesis, you could try disabling some plugins to see if that has any effect.
